fabric-entity-attribute-api-v1/src/main/java/net/fabricmc/fabric/impl/entity/attribute/FabricEntityAttributeRegistry.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
```

Comment:
It's 2020 @sfPlayer1 @modmuss50 we need to bump the license year.

Reply:
Not my fault, and this pr won't go out of scope.

Reply:
Yes it isn't really your issue, but player and modmuss need to bump this across the whole project.